04/index.html

```diff
@@ -1,32 +1,44 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    
+
     <nav>
-        <ul>
-            <li>
+        <button class="hamburger"></button>
+        <ul class="nav__list">
+            <li class="nav__item">
                 <a href="#">start</a>
             </li>
-            <li>
+            <li class="nav__item">
+                <a href="#">o nas</a>
+            </li>
+            <li class="nav__item">
+                <a href="#">cennik</a>
+            </li>
+            <li class="nav__item">
                 <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
+
+                <ul class="subMenu">
+                    <li class="nav__item"><a href="">strony internetowe</a></li>
                 </ul>
-                <ul>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+                <ul class="subMenuTwo">
+                    <li class="nav__item"><a href="">pozycjonowanie stron www</a></li>
                 </ul>
```

Comment:
zła konstrukcja, pisałem o tym na slacku. Powinien być tylko 1 `ul`, który zawiera 2 `li`

Na drugi poziom nie da się najechać bo od razu się chowa. Jest to spowodowane zbyt dużym odsunięciem elementu `ul`.